index.bs

```diff
@@ -580,8 +648,35 @@ with parameters |session| and |capabilities| is:
 
 </div>
 
+Common Data Types {#data_types}
+===============================
+
+This section defines data types which are common to many modules.
```

Comment:
These things don't seem like data types to me, they're types of targets, an abstract concept which is yet to be fleshed out. Since these things exist outside of this specification, maybe they can go in https://w3c.github.io/webdriver-bidi/#protocol, similar to what we've done with session? They do have a representation that we'll need to reuse, of course, maybe put that in the CDDL schema?

Reply:
I think that is in the schema. 

I don't think I want this stuff in #protocol because that's basically just the formalism whereas, as you say, these are about the representation in the schema of data types that can be transmitted over the wire. One option is just to put these things into a module like CDP does for everything; modules are going to need a way to define data types in general.

Reply:
Alright, ultimately this section defined only "browsing context id", "realm id" and the two snippets of CDDL. The two former seem more "mode" than "type", but I agree it's not a clean fit in #protocol either. Let's leave it here until it looks out of place.